test/lib/target-frameworks.test.ts

```diff
@@ -15,7 +15,7 @@ test('.Net Visual Basic project target framework extracted as expected', async (
     const targetFrameworks = await extractTargetFrameworksFromFiles(
         `${__dirname}/../fixtures/dotnet-vb-simple-project`,
         'manifest.vbproj');
-    t.deepEqual(targetFrameworks, [], 'targetFramework array is as expected');
+    t.deepEqual(targetFrameworks, ['v4.6.1'], 'targetFramework array is as expected');
```

Comment:
Could we please have one more test with more target frameworks separated by semicolon?

Reply:
Looks like for targetFrameworkVersion it is always 1 https://www.cyotek.com/blog/targeting-multiple-versions-of-the-net-framework-from-the-same-project